backend/sftp/sftp.go

```diff
@@ -58,6 +58,15 @@ func init() {
 		Name:        "sftp",
 		Description: "SSH/SFTP",
 		NewFs:       NewFs,
+		Config: func(ctx context.Context, name string, m configmap.Mapper, config fs.ConfigIn) (*fs.ConfigOut, error) {
```

Comment:
Can you do this in NewFs looking at the decoded Options?

The config callback is for oauth.